### PR TITLE
double the throughput on laboratory UCRs

### PR DESCRIPTION
**Pull request by emord**
Calls `self.config.get_all_values(doc)` once instead of twice for the UCRs. That is where all the db hits happen  to generate the rows.

This should be a huge boon to icds since all the UCRs in laboratory now. w=1 may be helpful for some of the review

familiar with UCR @NoahCarnahan @nickpell 

buddy @sravfeyn FYI @sheelio

### CODE REVIEW

File: corehq/apps/userreports/laboratory/adapter.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from sqlalchemy.exc import IntegrityError
```
Comment (dimagimon) [inline corehq/apps/userreports/laboratory/adapter.py]: F401 'IntegrityError' imported but unused

**Review by NoahCarnahan — APPROVED** [review]
Nice find

**Comment by emord** [comment]
@javierwilson @sravfeyn Not sure on the status of the nic deploy, but if this could get in that would be awesome

**Comment by javierwilson** [comment]
sure. we'll deploy soon, and we may even succeed....

**Comment by snopoke** [comment]
This is great, thanks!

**Pull request merged**